README.md

```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
+- :electric_plug: **A Highly modular plugin system:** making it possible to change almost anything.
 - :video_game: **Intuitive user experience:** managing output to allow the creation of loaders and dashboards.
-- :rocket: **Simple, programmatic API:** leveraging ES2015's async await with a minimal API.
+- :rocket: **Simple, pragmatic API:** leveraging ES2015's async await with a minimal API.
```

Comment:
It is actually a programmatic API (not a configuration)

Reply:
my bad :)